cumulus_library/studies/template/manifest.toml

```diff
@@ -1,8 +1,3 @@
-# If you'd like to run this template study as a proof of concept, you can load
```

Comment:
I deleted this since we're now copying it when you're creating a template. The equivalent instructions are now in first-time-setup.md